docker/android/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM ubuntu:zesty
+# sys_image can be overridden from the CLI
+ARG sys_image=debian:jessie
```

Comment:
As far as some quick research yields ... this feature was just not yet available at the time of the docker v1.12 releases. Do you have some restrictions that do not allow you to update to a more recent docker version ?
I am asking because I need to judge if supporting older docker versions will be important or not (I can think of a potential fix for backward compatibility, but that will be a rather cumbersome extension rather than a straightforward fix)